docs/api/schema.md

```diff
@@ -78,6 +78,33 @@ export type OrderOptions = (typeof orderSchema)[1];
 
 **Example:**
 
+```ts
+// Example with static defaults of const enums
+
+import { schema, types, VALIDATION_ACTIONS, VALIDATION_LEVEL } from 'papr';
+
+const statuses = ['processing', 'shipped'] as const;
+
+const orderSchema = schema(
+  {
+    _id: types.number({ required: true }),
+    user: types.objectId({ required: true }),
+    status: types.enum(statuses, { required: true }),
+  },
+  {
+    defaults: {
+      // const enums require the full type cast in defaults
+      status: 'processing' as (typeof statuses)[number],
```

Comment:
consider creating a separate `type` to use here, which is what most folks would do in their code.

e.g.
```
type Status = (typeof statuses)[number];
....
      status: 'processing' as Status,
```